Installation fails because of missing include for limits.h

**Describe the bug**
When installing with `pip install sktime`, I get:
```
   sktime/classification/shapelet_based/mrseql/SNode.cpp:89:39: error: ‘numeric_limits’ is not a member of ‘std’
       89 |             totalWildcardLimit = std::numeric_limits<int>::max();
          |                                       ^~~~~~~~~~~~~~
    sktime/classification/shapelet_based/mrseql/SNode.cpp:89:54: error: expected primary-expression before ‘int’
       89 |             totalWildcardLimit = std::numeric_limits<int>::max();
          |                                                      ^~~

```

**To Reproduce**
I have a standard, up to date, Arch Linux system

**Additional context**
This fixes the problem:
`CPLUS_INCLUDE_PATH=/usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/include-fixed/limits.h pip install sktime`

**Versions**
python: 3.9.5
gcc: 11.1.0

--- a/.all-contributorsrc
+++ b/.all-contributorsrc
@@ -1012,6 +1012,15 @@
         "bug",
       ]
     },
+    {
+      "login": "tombh",
+      "name": "Thomas Buckley-Houston",
+      "avatar_url": "https://avatars.githubusercontent.com/u/160835?s=80&v=4",
+      "profile": "https://github.com/tombh",
+      "contributions": [
+        "bug",
+      ]
+    },
   ],
   "projectName": "sktime",
   "projectOwner": "alan-turing-institute",

--- a/sktime/classification/shapelet_based/mrseql/SNode.h
+++ b/sktime/classification/shapelet_based/mrseql/SNode.h
@@ -13,6 +13,7 @@
 #include <vector>
 #include <algorithm>
 #include <iostream>
+#include <limits>
 
 class SNode
 {

